utils/doclint/documentation.js

```diff
@@ -322,11 +323,12 @@ Documentation.Member = class {
         continue;
       if (arg.langs.aliases && arg.langs.aliases[lang])
         arg.alias = arg.langs.aliases[lang];
-      arg.filterForLanguage(lang);
-      arg.type.filterForLanguage(lang);
-      if (arg.name === 'options' && !arg.type.properties.length)
+      const overriddenArg = (arg.langs.overrides && arg.langs.overrides[lang]) || arg;
+      overriddenArg.filterForLanguage(lang);
+      overriddenArg.type.filterForLanguage(lang);
```

Comment:
Drive-by: looks like we are already doing this in line 319, so drop this line.

Reply:
Done.